pom.xml

```diff
@@ -521,7 +521,38 @@
                 <scope>runtime</scope>
                 <version>${open-metadata.version}</version>
             </dependency>
+            <dependency>
+                <groupId>org.odpi.egeria</groupId>
+                <artifactId>ui-admin-services</artifactId>
```

Comment:
admin-services as a whole is unlikely to be a dependency for our code as we tend to be specific to api, server, spring etc, so wondering if this is needed. Also as it's a combionation of artifacts it's unlikely to be runtime only as some are server time (though I haven't looked in detail at your spring pieces)